prototype/mc_shortest_path/arduino_mc_shortest_path/arduino_mc_shortest_path.ino

```diff
@@ -0,0 +1,146 @@
+#include "arduino_serial_port.h"
+#include "monte_carlo.h"
+#include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define BOARD_LENGTH 3
+#define BOARD_SIZE 8 // 0 to 8
+
+void setup() {
+  // Initialize serial
+  Serial.begin(9600);
+}
+
+void loop() {
+  // Receive size of board
+  uint8_t nbValues;
+  readElement(&nbValues, sizeof(nbValues));
+
+  // Receive board
+  int8_t boardValues[nbValues];
+  for (uint8_t i = 0; i < nbValues; ++i) {
+    readElement(&boardValues[i], sizeof(int8_t));
+  }
+  uint8_t nPlayers;
+  readElement(&nPlayers, sizeof(nPlayers));
+
+  int minSimulation = 10;
+  int maxSimulation = 48;
+  int targetScore = 10;
+
+  Board board = {boardValues, nPlayers};
+  Game game = {isValidAction,
+               playAction,
+               getScore,
+               getPossibleActions,
+               getNumPossibleActions,
+               removeAction,
+               getBoardSize,
+               isDone,
+               0,
+               1};
+  Action action =
+      mcGame(board, 1, game, minSimulation, maxSimulation, targetScore);
+
+  // Return best action
+  writeElement(&action.xPos, sizeof(uint8_t));
+  writeElement(&action.yPos, sizeof(uint8_t));
+}
+
+uint8_t totalLength = 0;
+uint8_t initialBoard[9] = {1, 2, 3, 1, 2, 4, 5, 2, 1};
```

Comment:
Instead of beeing hardcoded could the initial board be sent by the user via the python file?